recipes/easylzma/all/conanfile.py

```diff
@@ -0,0 +1,54 @@
+import os
+from conans import ConanFile, tools, CMake
+
+class eazylzmaConan(ConanFile):
+    name = "easylzma"
+    license = "Public Domain"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/lloyd/easylzma"
+    description = ("An easy to use, tiny, public domain, C wrapper library around \
+                    Igor Pavlov's work that can be used to compress and extract lzma files")
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake"
+    topics = ("conan", "eazylzma", "lzma")
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    exports = ["LICENSE"]
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
+
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            self.options.remove("fPIC")
```

Comment:
```suggestion
           del self.options.fPIC
```

Same effect, but more pythonic